tests/e2e/test_rosacli_user.go

```diff
@@ -47,14 +49,20 @@ var _ = Describe("Edit User",
 					clusterAdminsUserName    = "testcu"
 				)
 
+				By("Try to list the user when there is no one")
+				_, output, err := userService.ListUsers(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).To(ContainSubstring(fmt.Sprintf("INFO: There are no users configured for cluster '%s'", clusterID)))
```

Comment:
`ContainSubstring` supports Sprintf, don't need to sprintf again
```suggestion
				Expect(textData).To(ContainSubstring"INFO: There are no users configured for cluster '%s'", clusterID))
```

Reply:
@xueli181114  Yes updated, Thanks